homework/english/Anastasia_Sheremet/Lecture1/Task_2.md

```diff
@@ -0,0 +1,60 @@
+Домашнее задание по лексике
+Напишите по 1 предложению к каждому слову из лексического блока в 1 уроке (слова есть в презентации).
+
+to boot
+He is booting updates to the server now.
```

Comment:
booting its like a starting the engine. Its can be confusing because to download and boot is the same in russian language

Reply:
May be you are right